### DISCUSSION TITLE
Hide Shorts, Playlists, and Livestreams in Related Videos?

### DISCUSSION THREAD

**Discussion by ghost** [ReVanced/revanced-discussions#1131] [Question] [answered]
A few months ago, the related_video custom filter stopped working, causing related videos to reappear under the main video. Currently, there is video_lockup$relatedH, which successfully hides "normal" videos but does not work for Shorts, playlists, or live streams. Does anyone know a way to hide these as well? Thanks in advance

**Comment by LisoUseInAIKyrios** [comment]
Using `video_lockup$relatedH` will cause the recommended videos to continuously reload in a loop while the video is open.  Battery life may be much worse and YT might flag your account as a bot.

The simpler solution is to either pull down on the video title and change to portrait fullscreen, or use the 'open videos fullscreen portrait' patch.

**Reply by princesingh4405** [reply]
From where do you guys know these custom filters like " video_lockup$relatedH "? I mean, is the any list of custom filters available somewhere?

**Reply by LisoUseInAIKyrios** [reply]
https://www.reddit.com/r/revancedapp/comments/1huxj08/comment/m5p2q9w

**Reply by oSumAtrIX** [reply]
It's unclear if YouTube changed or it's a different filter from what caused strain some time ago but it does not cause any strain anymore

**Comment by oSumAtrIX** [comment, marked as answer]
See https://www.reddit.com/r/revancedapp/comments/1qsv9x8/is_there_really_no_good_solution_to_related_videos/o2z13h2/
